crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -204,6 +214,8 @@ def restore_chef
           raise "Restore failed"
         end
 
+        # allow enable dns after restore.
+        disable_dns_path.delete if disable_dns_path.exist?
```

Comment:
> The idea was to delete the file before the chef-client is run. Thats why I put it outside cleanup().

okay thanks for the explanation. with the fixes mentioned above it looks good to me!

and if this fixes a bug can you please reference it in the commit? just add the (bsc#966419) and the link to bugzilla